Frameworks/Foundation/NSFileManager.mm

```diff
@@ -891,31 +1088,35 @@ - (NSString*)currentDirectoryPath {
 // Deprecated Methods
 
 /**
- @Status Stub
+ @Status NotInPlan
+ @Notes UWP does not provide an API to perform this
```

Comment:
you literally implemented this, sans handler support. Just Caveat it and ignore handler.